pkgs/top-level/all-packages.nix

```diff
@@ -16617,9 +16617,11 @@ in
   bluefish = callPackage ../applications/editors/bluefish {
     gtk = gtk3;
   };
-
+  
   bluejeans = callPackage ../applications/networking/browsers/mozilla-plugins/bluejeans { };
 
+  bluej-bin = callPackage ../applications/editors/bluej-bin { };
```

Comment:
You can probably just call this bluej as the non binary version isn't packaged, and if it were ever packaged it could be replaced.

Reply:
Will fix this, but let me note:

> We have a bad habit of not building java applications ourselves in nixpkgs (which would be nice to fix some day...) so what you've done is fine.

This is exactly why I named this -bin - as propaganda against the bad habit. I guess there is no motivation to check back on from-source build of this package otherwise.

Reply:
I appreciate the thought behind that.